calculator/parser.py

```diff
@@ -139,3 +182,96 @@ def rpn(tokens):
                 ",".join(str(t) for t in tokens)
             )
         )
+
+
+def infix(token_list):
+
+    """
+    Parses Infix notation and returns the equivilant RPN form
```

Comment:
Could you link to the wikipedia article on the shunting yard algorithm, please?

Reply:
This is still pending.

Reply:
👍